tests/foreman/cli/test_contentviews.py

```diff
@@ -187,19 +176,14 @@ def test_cv_create_cli_negative(self, test_data):
         system handles it gracefully
 
         """
+        try:
+            test_data['organization-id'] = make_org(cached=True)['id']
+        except CLIFactoryError as err:
+            self.fail(err)
 
-        org_obj = make_org(cached=True)
-
-        result = Org.info({'id': org_obj['id']})
-        self.assertEqual(result.return_code, 0, "Failed to create object")
-        self.assertEqual(
-            len(result.stderr), 0, "There should not be an exception here")
-
-        test_data['organization-id'] = org_obj['id']
         result = ContentView.create(test_data)
         self.assertNotEqual(result.return_code, 0)
```

Comment:
Nice catch. Actually this should be updated to:

```python
with self.assertRaises(CLIFactoryError):
    make_content_view(test_data)
```

After updating here is the test result with smoke flag equals 0:

```
$ py.test tests/foreman/cli/test_contentviews.py -ktest_cv_create_cli_negative
================================== test session starts ==================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 52 items

tests/foreman/cli/test_contentviews.py .......

================ 45 tests deselected by '-ktest_cv_create_cli_negative' =================
======================= 7 passed, 45 deselected in 57.37 seconds ========================
```